src/scss/custom.css

```diff
@@ -27,3 +28,27 @@
     --ifm-color-primary-lightest: #4fddbf;
     --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
 }
+
+.navbar--fixed-top {
+    backdrop-filter: blur(5px);
+    background-color: #ffffff60;
+}
+
+.navbar .navbar__item {
```

Comment:
why don't you use scss here?

```css
.navbar .navbar {
    &__item {
        @apply text-base font-semibold text-gray-700 hover:text-gray-500;
    }

    &__brand {
        @apply flex-row-reverse font-mono text-indigo-600 hover:text-indigo-400;
     
        .navbar__logo {
            @apply ml-1 size-6;
        }   
    }
}
```

Reply:
Because docusaurus had issues with our scss plugin as mentioned in the beginning of the file. Now that they've fixed this bug, I've updated docusaurus and we can use scss once again.